applications/models.py

```diff
@@ -31,6 +32,7 @@ class WinterStorageAreaChoice(models.Model):
 
     class Meta:
         unique_together = ("application", "priority")
+        ordering = ("application", "priority")
```

Comment:
I think, this should cause new migration files 🤔

Reply:
I'm not sure about it 🤔 🤔 because that's on the query level (i.e. the SQL that django builds) rather than on the DB schema.

Reply:
What I'm not sure about is `application` as an ordering key, since Application is a FK. Unless it would use the internal `int` id that django generates.

Reply:
yes, Django will use that `int`

Reply:
You are correct 👍  will generate them.

Reply:
Added the generated migration file. This seems to work in SQL level in a way that it generates following order clause: `ORDER BY "applications_winterstorageareachoice"."application_id" ASC, "applications_winterstorageareachoice"."priority" ASC` which makes sense.